README.md

```diff
@@ -7,7 +7,7 @@
 
 # pwa-lit-template
 
-##### [Getting started](#getting-started) | [Build for production](#build-for-production) | [Create a new page](#create-a-new-page) | [Browser support](#browser-support)
+##### [Getting started](#getting-started) | [Build for production](#build-for-production) | [Create a new page](#create-a-new-page) | [Configuration files](#configuration-files) | [Browser support](#browser-support)
```

Comment:
Personally y prefer just `Configuration` instead of `Configuration files`.

Reply:
Could be but maybe too generic? Maybe something more like `Environment configuration`?

Reply:
But it really is the only configuration we will have, right?

`Environment configuration` also works.

Reply:
Well not at all, without a context we could talk about the configuration of typescript or rollup to put an example (in fact, `configuration files` is even worse now that this example came to my mind 😂 ).

Ok, let's change it to `Environment configuration` it fits better.